api/gin/webServer.go

```diff
@@ -126,11 +126,15 @@ func (ws *webServer) StartHttpServer() error {
 		return err
 	}
 
-	log.Debug("starting web server",
-		"SimultaneousRequests", ws.antiFloodConfig.SimultaneousRequests,
-		"SameSourceRequests", ws.antiFloodConfig.SameSourceRequests,
-		"SameSourceResetIntervalInSec", ws.antiFloodConfig.SameSourceResetIntervalInSec,
-	)
+	if !ws.antiFloodConfig.WebServerAntifloodEnabled {
```

Comment:
why not replace L129-137 with:
```go
log.Debug("starting web server",
	"SimultaneousRequests", ws.antiFloodConfig.SimultaneousRequests,
	"SameSourceRequests", ws.antiFloodConfig.SameSourceRequests,
	"SameSourceResetIntervalInSec", ws.antiFloodConfig.SameSourceResetIntervalInSec,
	"with throttle middleware", ws.antiFloodConfig.WebServerAntifloodEnabled,
)
```

Reply:
It makes no point in printing SimultaneousRequests and so on if they are not effective

Reply:
oh, right